test/units/testsuite-17.11.sh

```diff
@@ -282,6 +282,8 @@ test_syntax_error 'KERNEL=="a|b", KERNEL=="c|d|e", NAME="f"' 'conflicting match
 # shellcheck disable=SC2016
 test_syntax_error 'ENV{DISKSEQ}=="?*", ENV{DEVTYPE}!="partition", ENV{DISKSEQ}!="?*", ENV{ID_IGNORE_DISKSEQ}!="1", SYMLINK+="disk/by-diskseq/$env{DISKSEQ}"' \
                   'conflicting match expressions, the line takes no effect'
+test_syntax_error 'ACTION=="a*", ACTION=="bc*", NAME="d"' 'conflicting match expressions, the line takes no effect'
```

Comment:
"takes no effect" is strange. (We use "take" when talking about options, e.g. "Option A= takes true and false".)
"has no effect" is more natural. Maybe change the text here and in the preexisting cases?

Reply:
```
$ git grep 'takes no effect' src
src/udev/udev-rules.c:                log_line_warning(rule_line, "The line takes no effect, ignoring.");
src/udev/udev-rules.c:                                log_line_notice(line, "The line takes no effect any more, dropping");
src/udev/udev-rules.c:                                log_line_error(line, "conflicting match expressions, the line takes no effect");
```

The `line takes no effect` wording was introduced about 4 years ago by commit 25de7aa7b90.
I don't mind rewording, but this would have to be a separate commit.
@yuwata, what do you think about it?